userspace/libsinsp/threadinfo.cpp

```diff
@@ -360,6 +360,7 @@ void sinsp_threadinfo::init(scap_threadinfo* pi)
 
 	m_comm = pi->comm;
 	m_exe = pi->exe;
+	m_exepath = pi->exepath;
```

Comment:
You mostly need to manually handle serialization in `scap_write_proclist_entry` and deserialization in `scap_read_proclist`, you can look at how things are done for the other string fields, it's mostly boilerplate.

Alternatively, if you don't want to do that, just make sure that in `scap_read_proclist` you explicitly initialize `exepath` by writing a `\0` to it or something, so sinsp doesn't read garbage.